.lintstagedrc

```diff
@@ -2,7 +2,7 @@
 	"*.coffee": [
 		"resin-lint"
 	],
-	"*.ts": [
+	"*.{d.ts,ts}": [
```

Comment:
`*.ts` should match both `abc.ts` and `xyz.d.ts`, so I don't think this change is necessary, could you recheck/point to docs where it's failing?